react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -413,6 +416,25 @@ class Thumbnail extends Component<Props, State> {
      */
     componentDidMount() {
         this._onDisplayModeChanged();
+
+        // Listen to track streaming status changed event to keep it updated.
+        const { _videoTrack, dispatch } = this.props;
+        if (_videoTrack && !_videoTrack.local) {
+            _videoTrack.jitsiTrack.on(JitsiTrackEvents.TRACK_STREAMING_STATUS_CHANGED, this.handleTrackStreamingStatusChanged);
+            dispatch(trackStreamingStatusChanged(_videoTrack.jitsiTrack, _videoTrack.jitsiTrack.getTrackStreamingStatus?.()));
```

Comment:
I wonder if it is better to be less defensive here and just let it fail if `getTrackStreamingStatus` is undefined. It might not be clear why a tile is showing an avatar

Reply:
In what case `getTrackStreamingStatus` is `undefined`?

Reply:
changed